src/Directory.Build.targets

```diff
@@ -9,8 +9,10 @@
 
   <!-- Configure trimming for projects on .NET 6 and above -->
   <PropertyGroup Condition="'$(TargetFramework)' == 'net6.0' OR '$(TargetFramework)' == 'net7.0'">
-    <EnableTrimAnalyzer>true</EnableTrimAnalyzer>
     <IsTrimmable>true</IsTrimmable>
+    <EnableTrimAnalyzer>true</EnableTrimAnalyzer>
```

Comment:
Technically unnecessary as `IsTrimmable` implies this.

Reply:
Keeping them for now to be explicit, but will definitely be doing another PR once https://github.com/dotnet/sdk/issues/31284 is available and just use that new `IsAotCompatible` property, looks super nice! Looking forwards to it 😄